src/shutdown/umount.c

```diff
@@ -352,16 +352,30 @@ static int md_list_get(MountPoint **head) {
         if (r < 0)
                 return r;
 
+        r = sd_device_enumerator_add_match_property(e, "DEVTYPE", "disk");
```

Comment:
a comment would be good explaining what this is supposed to ensure, i.e. filter out partitions